tests/channels/clone_channel.ts

```diff
@@ -0,0 +1,48 @@
+import { tempData } from "../ws/start_bot.ts";
+import { assertEquals, assertExists } from "../deps.ts";
+import { cache } from "../../src/cache.ts";
+import { cloneChannel } from "../../src/helpers/channels/clone_channel.ts";
+import { delayUntil } from "../util/delay_until.ts";
+
```

Comment:
Please add a `ifItFailsBlameWolf` function use the other tests for reference.

Reply:
i think its fine if it just uses 1 test since u can't reuse the code here